docs/fundamentals/week-3/coursework.md

```diff
@@ -20,3 +20,48 @@ Share one of your projects in Google Classroom, not forgetting to include:
 - Your deployed website
 
 Share your deployed website in your cohort's Slack channel.
+
+## Personal development - It's time to do your research  (1h)
+
+Task 1: 
+
+Set up your LinkedIn profile and let employers know that you have started your journey into the tech industry and have  built your first product! 
+
+Task 2: 
+
+It’s time to do your research, it’s time to get familiar with the industry that you want to work in. Go on google and search for roles, look at the **person specifications** that companies are hiring for, please fill out the form below with the **three soft skills** employers are currently looking for, you will be able find roles on CW Jobs, Linkedin, Indeed and any jobs board: 
```

Comment:
```suggestion
It’s time to do your research. It’s time to get familiar with the industry that you want to work in. Search for roles on Google: look at the **person specifications** that companies are hiring for.

Fill out the form below with the **three soft skills** employers are currently looking for. 

You will find roles on [CW Jobs](https://www.cwjobs.co.uk/), [Linkedin](https://www.linkedin.com/),[ Indeed](https://uk.indeed.com/),  and any jobs board.
```

(comma splices)

Reply:
...actually, I'd use a semi colon here :  Search for roles on Google; look at the **person specifications** that companies are hiring for.